pkg/defaults/defaults.go

```diff
@@ -49,7 +48,6 @@ var Defaulters = []Defaulter{
 	docker.Pipe{},
 	artifactory.Pipe{},
 	s3.Pipe{},
-	blob.Pipe{},
```

Comment:
merge conflict still? 

this should be here...

Reply:
fixed in 5155302